client/src/pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from "react";
+import CredentialsInputBox from "../../components/CredentialsInputBox/CredentialsInputBox.js";
+import {
+  HeaderText,
+  SubtitleText,
+} from "../../components/ThemeText/ThemeText.js";
+import { CredentialsButton } from "../../components/CredentialsButton/CredentialsButton.js";
+import { DescriptiveTextButton } from "../../components/DescriptiveTextButton/DescriptiveTextButton.js";
+import { useNavigate } from "react-router-dom";
+
+import "./LoginPage.css";
+
+const LoginPage = () => {
+  const navigate = useNavigate();
+  const routeToSignUp = () => {
+    navigate("signup");
+  };
+
+  useEffect(() => {
+    document.body.style.background = "#032A5C";
```

Comment:
The background for the body can be placed in the index.css file (found in the root directory)
The main purpose of useEffect is for any initial renders, subsequent renders due to states being updated or cleanup

Reply:
Fixed: background color is now set through the top-most `div` in `CredentialsPage`.